receiver/mongodbreceiver/metadata.yaml

```diff
@@ -238,3 +256,39 @@ metrics:
       aggregation: cumulative
       monotonic: false
     attributes: []
+  mongodb.locks.acquire_count:
+    description: Number of times the lock was acquired in the specified mode.
+    unit: "{count}"
+    enabled: true
+    sum:
+      value_type: int
+      aggregation: cumulative
+      monotonic: false
```

Comment:
Based on the description, I think this is likely monotonic. Does it ever decrease (aside from a rollover or reset)?

Reply:
you are right, updated the code.